src/wayland/seat/pointer.rs

```diff
@@ -5,7 +5,7 @@ use wayland_server::protocol::{wl_pointer, wl_surface};
 // TODO: handle pointer surface role
 
 struct PointerInternal {
-    known_pointers: Vec<wl_pointer::WlPointer>,
+    known_pointers: Vec<(wl_pointer::WlPointer, i32)>,
```

Comment:
Is there a big need to to track the versions here, given we can just to `pointer.version()` whenever we need ?

The pointer automatically inherit the same version as the seat that created them.

Reply:
Oh I did not know that. `pointer.version()` would be sufficient in that case.